README.md

```diff
@@ -35,40 +137,89 @@ The raw output looks something like this:
     }
 ```
 
-# Installation
+### Further reading
 
-#### Install from source
+- For more info about analytics codes and what the GA-4 rollout means for OSINT: [https://digitalinvestigations.substack.com/p/what-the-rollout-of-google-analytics]
 
-1. Clone repo:
+- For an example investigation usings analytics codes: [https://www.bellingcat.com/resources/how-tos/2015/07/23/unveiling-hidden-connections-with-google-analytics-ids/]
 
-```terminal
-git clone git@github.com:jclark1913/osint-google-analytics.git
-```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-2. Navigate to root, create a venv and install requirements.txt:
 
-```terminal
-cd osint-google-analytics
-python -m venv venv
-source venv/bin/activate
-pip install -r requirements.txt
-```
+### Built With
+
+<div align="center">
+
+![Python][Python]
+![Pandas][Pandas]
+
+</div>
+
+Additional libraries/tools: [BeautifulSoup4](https://pypi.org/project/beautifulsoup4/), [Asyncio](https://docs.python.org/3/library/asyncio.html), [Aiohttp](https://docs.aiohttp.org/en/stable/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+## Installation
+
+### Install with pip
```

Comment:
```suggestion
### Install from [pypi](https://pypi.org/project/wayback-google-analytics/) (with pip)

[![PyPI][pypi-shield]][pypi-url]
```

Hope I got the shield thing right with the links at the end, made 2 suggestions.